liberbatch.py

```diff
@@ -1,19 +1,15 @@
 import os, meichecker, liberupdatev5

```

Comment:
Are you using a code linter? This is not PEP8 compliant (https://peps.python.org/pep-0008/#imports)